### PR TITLE
Reframe the InCommon host cert options

### PR DESCRIPTION
**Pull request by brianhlin**
I'd like the documentation to focus on use cases instead of tools. Plus some other misc focuses.

Once we get this PR approved and merged, I'll approve and merge: https://github.com/opensciencegrid/docs/pull/537

### CODE REVIEW

**Comment by brianhlin** [comment]
@jteheran can you review and merge this?

File: docs/security/host-certs.md [hunk 1]
```diff
@@ -125,49 +125,56 @@ generated above.
...
-        
-        Extracting the private key from the .p12 file
-        :::console
-        user@host $ openssl pkcs12 -in incommon_file.p12 -nocerts -out ~/path_to_dir/incommon_user_key.pem
-        
-        Extracting the certificate from the .p12 file
-        :::console
-        user@host $ openssl pkcs12 -in incommon_file.p12 -nokeys -out ~/path_to_dir/incommon_user_cert.pem
+   (e.g. [UW-Madison InCommon contact](https://it.wisc.edu/about/office-of-the-cio/cybersecurity/security-tools-software/server-certificates/)),
+1. Request a Department Registration Authority user with SSL auto-approve enabled and client certificate:
```
Comment (jteheran) [inline docs/security/host-certs.md]: Request a Department Registration Authority user with SSL auto-approve enabled and **a** client certificate:

File: docs/security/host-certs.md [hunk 2]
```diff
@@ -125,49 +125,56 @@ generated above.
...
+    - If they do not grant your request, you will not be able to request, approve, and retrieve certificates yourself.
+      Instead, you must [request certificates from your RA](#requesting-certificates-from-a-registration-authority).
+    - If they grant your request, you will receive an email with your client certificate.
```
Comment (jteheran) [inline docs/security/host-certs.md]: If they grant your request, you will receive an email with instructions on how to request your client certificate.

File: docs/security/host-certs.md [hunk 3]
```diff
@@ -125,49 +125,56 @@ generated above.
...
                     --pkey ~/path_to_dir/incommon_user_key.pem \
                     --hostname <HOSTNAME> \
                     --altname <ALTNAME> \
                     --altname <ALTNAME2>
 
-        Requesting certificates in bulk using a hostfile name
+- Requesting certificates in bulk using a hostfile name:
+
         :::console
         user@host $ osg-incommoncert-request --username <INCOMMONLOGIN> \
```
Comment (jteheran) [inline docs/security/host-certs.md]: user@host $ osg-incommoncert-request --username **<INCOMMON_LOGIN>**

**Review by jteheran — CHANGES_REQUESTED** [review]
I left three comments.

**Comment by brianhlin** [comment]
@jteheran alright, comments should be addressed. Just an FYI for the future, you can add "suggestions" to the PR so that the author can accept them directly via the GitHub interface (https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request)

**Comment by jteheran** [comment]
That's what I thought I did. I added line comments. Not sure what went wrong.

**Comment by brianhlin** [comment]
When you're adding an in-line comment, there's a button in the UI that turns it into a suggestion, which should add something that looks like the following to the text box:

    ```suggestion
    <original text here>
    ```
Does the PR look good now? Can you merge it?

**Pull request merged**